src/taoverse/model/model_updater.py

```diff
@@ -46,7 +46,13 @@ def verify_model_satisfies_parameters(
 
         # Check that the parameter count of the model is within allowed bounds.
         parameter_size = sum(p.numel() for p in model.pt_model.parameters())
-        if parameter_size > model_constraints.max_model_parameter_size:
+        bt.logging.debug(f'Number of model parameters is {parameter_size}')
```

Comment:
Move this log inside the if to avoid noise? And change the max param allowed log to say parameters must be between min and max for this competition.